### PR TITLE
Homework5 for lesson6

### PR DESCRIPTION
**Pull request by HelensBadCode**

### CODE REVIEW

File: jsBasics/homeWork5/task2.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function isPersonAdult(age){
+    let result = age >= 18;
```
Comment (sttaran) [inline jsBasics/homeWork5/task2.js]: why let but not const?

File: jsBasics/homeWork5/task1.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    return result;
+}
+console.log("The area of the rectangle :",rectangleAreaExpression(26, 32));
+//----------------------------------------------------------------------------------------------------------------------
+
+//arrow function expression
+const rectangleAreaArrow = (width, height) => {
+    let result = width * height;
+    return result;
+}
```
Comment (sttaran) [inline jsBasics/homeWork5/task1.js]: why let but not const?

Comment (sttaran) [reply jsBasics/homeWork5/task1.js]: function name should represent and action

**Review by sttaran — CHANGES_REQUESTED** [review]

File: jsBasics/homeWork5/task1.js (hunk 2 above)
Comment (HelensBadCode) [reply jsBasics/homeWork5/task1.js]: @sttaran I didn't find in our lesson that I can't use let in functions. But if points are deducted for this, then I corrected it.

File: jsBasics/homeWork5/task2.js (hunk 1 above)
Comment (HelensBadCode) [reply jsBasics/homeWork5/task2.js]: @sttaran I didn't find in our lesson that I can't use let in functions. But if points are deducted for this, then I corrected it.

File: jsBasics/homeWork5/task1.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+//function declaration
+function calculationAreaOfRectangle(width, height){
```
Comment (sttaran) [inline jsBasics/homeWork5/task1.js]: `calculateRectangleArea ` would be better name

function name should answer the question "what to do?"

**Review by sttaran — APPROVED** [review]

**Pull request merged**